src/main/java/com/univapay/sdk/types/brand/QrCpmBrand.java

```diff
@@ -26,6 +26,7 @@ public enum QrCpmBrand {
   GCash("gcash"),
   Dana("dana"),
   TrueMoney("truemoney"),
+  HelloMoney("hello_money"),
```

Comment:
HelloMoneyなら、返却される値は、`hellomoney`です

```suggestion
  HelloMoney("hellomoney"),
```